### PR TITLE
Removing webapp up command & max_cli_version

### PR DESCRIPTION
**Pull request by panchagnula**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

**Comment by panchagnula** [comment]
@tjprescott  is there a way to restart the travis-ci build - may be i don't have permission to trigger the build. I update whl file & want to trigger a new build.

File: src/index.json [hunk 1]
```diff
@@ -2120,11 +2120,10 @@
         ],
         "webapp": [
             {
-                "downloadUrl": "https://github.com/patricklee2/extensions-whl/raw/master/webapp-0.2.17-py2.py3-none-any.whl",
-                "filename": "webapp-0.2.17-py2.py3-none-any.whl",
+                "downloadUrl": "https://github.com/panchagnula/azure-cli-extensions/raw/sisirap-extensions-whl/dist/webapp-0.2.18-py2.py3-none-any.whl",
+                "filename": "webapp-0.2.18-py2.py3-none-any.whl",
                 "metadata": {
                     "azext.isPreview": true,
-                    "azext.minCliCoreVersion": "2.0.46",
```
Comment (tjprescott) [inline src/index.json]: Why are you removing this?

Comment (tjprescott) [inline src/index.json]: Have you talked to @limingu regarding adding maxCliCoreVersion 2.0.66?

File: src/webapp/azext_webapp/__init__.py [hunk 2]
```diff
@@ -24,24 +17,12 @@ def __init__(self, cli_ctx=None):
```
Comment (tjprescott) [inline src/webapp/azext_webapp/__init__.py]: You will need to remove the `min_profile` annotation here for CLI core version 2.0.67 and up or your extension will break (that property is being removed). You will need an index to that last working extension for 2.0.46-2.0.66. Please talk to @limingu.

File: src/index.json (hunk 1 above)
Comment (panchagnula) [reply src/index.json]: this is done.

**Comment by panchagnula** [comment]
@tjprescott can you help me figure out the failure in the travis-ci. thank you.

File: src/webapp/azext_webapp/azext_metadata.json [hunk 3]
```diff
@@ -1,5 +1,4 @@
 {
     "azext.maxCliCoreVersion": "2.0.66",
-    "azext.minCliCoreVersion": "2.0.46",
```
Comment (tjprescott) [inline src/webapp/azext_webapp/azext_metadata.json]: Why are you removing the minCliCoreVersion?

File: src/webapp/azext_webapp/__init__.py (hunk 2 above)
Comment (tjprescott) [reply src/webapp/azext_webapp/__init__.py]: This is still not done.

**Comment by panchagnula** [comment]
@tjprescott , made the updates - but the CI/CD is still failing - one of the errors is regarding missing help - but the help.py exists -  help please.

**Comment by panchagnula** [comment]
@limingu can you help restart this build please?

**Comment by panchagnula** [comment]
@tjprescott & @limingu i will merge this once i get a sign-off.

**Pull request merged**